RPGManager_development/src/rpg_database/character_sheet/Background.java

```diff
@@ -69,6 +88,7 @@ public void setSelfInSheet(CharacterSheet characterSheet) {
 					Fields.BASECLASS).toString()));
 		}
 		characterSheet.characterData.put(Fields.BACKGROUND, this);
+		characterSheet.setData(Fields.LANGUAGES, this.language);

```

Comment:
Oks, na a telefon után meggyőztél. Akkor itt az a megoldás hogy minden alkalommal amikor új backgroundot állítunk be, törlöd a LanguagesSet-jét a CharacterSheet-nek, és újra feltöltöd az adott background nyelveivel